examples/cifar10/train_full.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env sh
 
+# Check if TOOLS is unset
+if [ -z ${TOOLS+x} ];
+then
 TOOLS=./build/tools
+fi
```

Comment:
Done. Changed as below -
if [ -z "$TOOLS" ]; then
    TOOLS=./build/tools
fi
Just instead of [[]], I used [] as the script has /bin/sh as a first line. [[]] is valid for bash.